.travis.yml

```diff
@@ -8,6 +8,10 @@ before_install:
 - git remote set-branches origin master && git fetch
 - git checkout master
 - git checkout -
+
+after_install:
+- npm run nsp-check # -- --warn-only
```

Comment:
Similar to the empty `exceptions` Array above, we could add `--warn-only` flag here if we want `nsp` to always exit with an exit code of 0 (temporarily or permanently).